ui/plugin-lifecycle/src/main/interfaces/Plugin.ts

```diff
@@ -0,0 +1,58 @@
+export interface Plugin {
```

Comment:
many of these objects should also be available in @vcd/bindings...PluginUploadSpec, UiPluginMetadata, etc.

Reply:
I can't find any class or interface with this names.

Reply:
I had rebuild the packages folder and it's okey.